test/extended/clusterversion/OWNERS

```diff
@@ -0,0 +1,5 @@
+reviewers:
+  - cluster-version-operator-test-case-reviewers
```

Comment:
Our existing consumer for this alias is [in `test/extended/util/openshift/clusterversionoperator`][1].  I'm not all that clear on how `test/extended` is organized, but it seems like we should keep these CVO-functionality tests collected in the same directory?

[1]: https://github.com/openshift/origin/tree/94a4d2a6f202f64f89a1b747b44484c358f8299d/test/extended/util/openshift/clusterversionoperator

Reply:
that file seems to be the utils file whereas [this](https://github.com/openshift/origin/blob/94a4d2a6f202f64f89a1b747b44484c358f8299d/test/extended/adminack/adminack.go#L25) is where the test is actually defined, listing cvo folks as [owners](https://github.com/openshift/origin/blob/94a4d2a6f202f64f89a1b747b44484c358f8299d/test/extended/adminack/OWNERS#L4). This is similar.